include/seqan3/io/sequence_file/output.hpp

```diff
@@ -426,12 +426,7 @@ class sequence_file_output
         requires tuple_like<std::ranges::range_reference_t<rng_t>>
     //!\endcond
     {
-    #if defined(__GNUC__) && (__GNUC__ == 9) // an unreported build problem of GCC9
-        for (auto && record : range)
-            f.push_back(std::forward<decltype(record)>(record));
-    #else // ^^^ workaround | regular solution ↓↓↓
         f = range;
-    #endif
         return std::move(f);
```

Comment:
🤷 
It is definitely needed, because I tried to remove it :D
Copy elision doesn't take effect here.
I think this is some code where we have overloads for each `const` `&` `&&` combo?